src/main/kotlin/lotto/domain/LottoTickets.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.domain
+
+import lotto.dto.WinStats
+import lotto.util.AutoNumbers
+
+class LottoTickets(private val money: Money, autoNumbers: AutoNumbers) {
+    val lottoTickets: List<LottoTicket> = List(money.countLotto()) {
+        LottoTicket(autoNumbers.generateNumbers())
+    }
+
+    fun getWinStats(winNumbers: LottoTicket): WinStats {
+        var amount: Long = 0
+
+        val matchMap: Map<WinResult, Int> = lottoTickets.filter {
+            it.getWinResult(winNumbers) !== WinResult.LOSE
+        }.groupingBy {
+            it.getWinResult(winNumbers)
+        }.eachCount()
```

Comment:
아래와 같이 리팩토링 하면 더 깔끔해질 것 같아요 :) 

```suggestion
        val matchMap: Map<WinResult, Int> = lottoTickets.map { it.getWinResult(winNumbers) }
            .filter { it !== WinResult.LOSE }
            .groupingBy { it }
            .eachCount()
```